tests/log_fidelity/test_bgp_shutdown.py

```diff
@@ -0,0 +1,29 @@
+import logging
+import pytest
+
+from tests.common.plugins.loganalyzer.loganalyzer import LogAnalyzer, LogAnalyzerError
+
+logger=logging.getLogger(__name__)
+
+pytestmark = [
+    pytest.mark.topology('any')
+]
+
+expected_log = "admin state is set to 'down'"
```

Comment:
This expected string can be made more strict for bgp matches:
`Apr 14 22:26:53.660081 vlab-01 INFO bgp#bgpcfgd: Peer 'default|10.0.0.63' admin state is set to 'down'`

Something like:
`^.*bgp#bgpcfgd: Peer.*admin state is set to 'down'`

Reply:
Similarly, have you checked if the expected string remains same for 201911 image too?

Reply:
I just checked with 201911 and the expected string remains the same!